packages/data-store/package.json

```diff
@@ -13,6 +13,7 @@
     "typeorm-postgres:migration:run": "pnpm run typeorm -- migration:run -c migration-postgres"
   },
   "dependencies": {
+    "@sphereon/ssi-sdk.core": "workspace:*",
```

Comment:
Pleases also update the references in tsconfig.json

Reply:
added core to tsconfig of data-store